.github/workflows/test.yml

```diff
@@ -2,22 +2,21 @@ name: Test
 
 on:
   pull_request:
-    branches: [ master ]
+    branches: [master]
```

Comment:
Consider using 'main' instead of 'master' if your repository has transitioned to using 'main' as the default branch name.